### DISCUSSION TITLE
Suggestion to add Favicon in the website

### DISCUSSION THREAD

**Discussion by SoumyadeepOSD** [networkx/networkx#6435] [General]
## Hey everyone, I am new here👋 
Just started learning the GNN by using NetworkX, it's an awesome library! 
While visiting the website  https://networkx.org/documentation/stable/   I found there is no favicon while opening the website

![image](https://user-images.githubusercontent.com/115442240/219102836-fafe8a8e-ab3a-43e4-9ba7-b80646d8daf3.png)

Should we add Favicon in the website?

**Comment by rossbar** [comment]
This should be straightforward to implement, see: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/branding.html#add-favicons

**Reply by SoumyadeepOSD** [reply]
@rossbar  Can I contribute to it?

**Reply by rossbar** [reply]
Please do!

**Reply by SoumyadeepOSD** [reply]
> Please do!

I've installed all required dependencies, can you tell me how can I run the website in localhost?

**Reply by rossbar** [reply]
The documentation is generated using `sphinx`. The result is a static site, so there's no need to run a web server. Assuming you're on a *nix system with all the dependencies installed, you can build the docs from the `doc` directory with:

```bash
make html
```

You can then view the html directly in a browser, e.g. `firefox build/html/index.html`

**Comment by jarrodmillman** [comment]
See #7043.
